en/docs/get-started/tutorials/create-your-first-integration.md

```diff
@@ -0,0 +1,134 @@
+# Create Your First Integration
+
+An integration is a component with logic to integrate APIs, microservices, applications, and data in different languages and formats.
```

Comment:
Shall we say, that you can use an integration component to integrate APIs, microservices, application, and data  in any language and format.

Reply:
Done